__tests__/__system__/cli/disable/urimap/cli.disable.urimap.system.test.ts

```diff
@@ -73,8 +73,32 @@ describe("CICS disable urimap command", () => {
         expect(output.stdout.toString()).toContain("success");
         await sleep(sleepTime);
 
+        output = runCliScript(__dirname + "/../../install/urimap/__scripts__/install_urimap.sh", TEST_ENVIRONMENT,
+        [urimapName, csdGroup, regionName]);
+        stderr = output.stderr.toString();
+        expect(stderr).toEqual("");
+        expect(output.status).toEqual(0);
+        expect(output.stdout.toString()).toContain("success");
+        await sleep(sleepTime);
```

Comment:
For future reference, feel free to create functions that help you write/ copy-paste less stuff 😝 
No change requested!